### PR TITLE
Allow up to 999 clips to be saved

### PR DESCRIPTION
**Pull request by hungri-yeti**
- increased width of Remember and Favorites text fields in preferences, changed corresponding steppers to max 999
- adjusted label font size and positioning of related preferences UI elements
- increased width of clip count field in bezel

### CODE REVIEW

**Comment by diegodesouza** [comment]
I wonder why these pull requests aren't being reviewed or merged. Is this repo dead?

**Comment by knyttl** [comment]
@hungri-yeti would you have a build with your changes?

**Comment by ptoche** [comment]
for sure 99 clips isn't enough, 999 may be more like it. However, the feature to save the clips to disk makes it >> 999. The only problem is that the clips are saved to the desktop with no apparent option to move the directory elsewhere...

**Comment by vfonic** [comment]
Yes, please! I came here to look for a solution to having more than 99 clips!

I'll try to figure out if I can fix the breaking changes.

**Comment by hungri-yeti** [comment]
@MarkJerde I know this PR is very stale but before I close it just curious why it was never approved?

**Comment by MarkJerde** [comment]
@hungri-yeti I’ve never approved or merged PRs on this project. I guess I have access to do so, though, so it’d be better for me to step up than have the project die or fork. Please leave the PR open for now. 

Before I take a look, though, could you search the discussions on here and let me know what discussion of the limit there has been in the past, and then tag TermiT in a comment asking his thoughts on the change since he owns the project?

I certainly can understand the desire for the history capacity to be increased, and assume there isn’t a performance concern since you’ve probably been using this for… years. I just don’t see fit for me to unilaterally accept changes that could change the scopes or vision of the project. With TermiT’s assent of the spirit of the change, I’m more than capable of ensuring the changes are non-breaking, though. And if he isn’t available to weigh in on such things, I could probably figure out a team of stakeholders to avoid unilateral decisions. 

Sorry for the delay to you contributions, and I’ll keep an eye on this, because it deserves an up or down vote for sure. 

Thanks,
Mark

**Comment by knyttl** [comment]
>  the scopes or vision of the project. 

is there any vision of the project? can we read it somewhere?

**Comment by MarkJerde** [comment]
> > the scopes or vision of the project.
> 
> is there any vision of the project? can we read it somewhere?

Flycut is fully contained within this GitHub project, so scope and vision exist as established by files and comments in the repository and comments on PRs and issues on GitHub. There aren't any private discussions or documents, so everyone can see it all and contribute to discussions. It's easy to create a PR so someone could add any documentation files they think would help, and as long as the files are correct I wouldn't have an issue with the PR (or with a PR that later deletes them if they have become incorrect and unmaintained, as the mutability of a repository nicely allows temporal content to come and go without fear of the long-term). GitHub issues also make a good platform for feature suggestions and discussions of scope and vision.

File: UI/BezelWindow.m [hunk 1]
```diff
@@ -183,7 +183,7 @@ -(NSRect) textFrame
 
 -(NSRect) charFrame
 {
-    return NSMakeRect(([self frame].size.width - (3 * lineHeight)) / 2, 7, 4 * lineHeight, 1.2 * lineHeight);
+    return NSMakeRect(([self frame].size.width - (3 * lineHeight)) / 2, 7, 8 * lineHeight, 1.2 * lineHeight);
```
Comment (MarkJerde) [inline UI/BezelWindow.m:186]: It looks like with `4 * lineHeight` (on a tangent about code that was already there before you, using height constants to define width? Ew. 😄) provides 8px leading and trailing margins for `99 of 99`. A `9` is 8px wide (with kerning), and `lineHeight` is `16`. This change adds 64px of width to provide space for at most two 9s, which only need a total 16 px. I haven't played with running the code, but it seems based on this that either the shaded area behind the count is now really wide (looking a bit odd) or that shading is provided by something else and this text field is now a lot wider than the shading area (which opens a door to other problems beyond the 8px margin being eaten to nothing). Is this assessment correct?

**Pull request closed without merging**